tests/conftest.py

```diff
@@ -23,23 +23,16 @@ class TestCollectArgs(Process):
     def __init__(self):
         inputs = [
             ComplexInput(
-                "local_file",
-                "Local file",
-                max_occurs=5,
-                abstract="Path to the local input file",
-                supported_formats=[FORMATS.NETCDF],
-            ),
-            ComplexInput(
-                "opendap_url",
-                "OPeNDAP URL",
-                max_occurs=5,
-                abstract="URL to the opendap input file",
-                supported_formats=[FORMATS.DODS],
+                "file",
+                "File",
+                max_occurs=3,
```

Comment:
Thinking about the usage of the input, I think the `max_occurs` should be 1 unless you want to use it differently.